### PR TITLE
SK-1764-WEB-Email-Zimbra-mobile-app 

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -1587,6 +1587,7 @@
             + [Using Zimbra webmail](web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra)
             + [FAQ Zimbra OVHcloud](web_cloud/email_and_collaborative_solutions/mx_plan/faq-zimbra)
             + [Configure Zimbra account on email client](web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps)
+            + [Configure Zimbra account on Zimbra app](web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios)
```
Comment (Y0Coss) [inline pages/index.md]:
```suggestion
            + [How to configure a Zimbra email account on the Zimbra mobile application](web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios)
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+---
+title: Zimbra - Configurer son compte e-mail sur l'application Zimbra
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]:
```suggestion
title: "Zimbra - Configurer son compte e-mail sur l'application mobile Zimbra"
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+excerpt: Découvrez comment configurer votre adresse e-mail Zimbra sur l'application Zimbra
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]:
```suggestion
excerpt: "Découvrez comment configurer votre adresse e-mail Zimbra sur l'application mobile Zimbra disponible sur Android et iOS"
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+updated: 2024-11-06
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]:
```suggestion
updated: 2024-11-07
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+> [!warning]
+>
+> OVHcloud met à votre disposition des services dont la configuration, la gestion et la responsabilité vous incombent. Il vous revient de ce fait d'en assurer le bon fonctionnement.
+>
+> Nous mettons à votre disposition ce guide afin de vous accompagner au mieux sur des tâches courantes. Néanmoins, nous vous recommandons de faire appel à un [partenaire spécialisé](https://marketplace.ovhcloud.com/c/support-collaboration) et/ou de contacter l'éditeur du service si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section « Aller plus loin » de ce guide.
+
+## Prérequis
+
+- Disposer d’une adresse e-mail Zimbra.
+- Avoir installé l'application Zimbra sur votre appareil Android ou iOS.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]: shouldn't we provide links to ios store and google play store ?

- https://play.google.com/store/apps/details?id=com.zimbra.modernapp&hl=fr
- https://apps.apple.com/cm/app/zimbra-email-collaboration/id1554848550

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+## En pratique
+
+### Configurer son adresse e-mail
+
+Suivez les étapes d'installation en cliquant sur les onglets ci-dessous :
+
+> [!tabs]
+> **Etape 1**
+>>
+>> lorsque vous lancez l'application pour la première fois, la fenêtre ci-dessous s'affiche. Saisissez l'URL du serveur Zimbra OVHcloud `https://zimbra1.mail.ovh.net`, appuyez sur `Continuez`{.action} pour valider.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]: is the button "Continuez" or "Continuer" ?

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+>>
+> **Etape 2**
+>>
+>> Saisissez votre **adresse e-mail** Zimbra et le  **mot de passe** associé, appuyez ensuite sur `Connexion`{.action}
+>>
+>> ![zimbra_app](images/zimbra_app_connect02.png){.thumbnail .w-600 .h-600}
+>>
+> **Etape 3**
+>>
+>> Vous avez maintenant accès au contenu de votre compte e-mail. Par défaut c'est votre boite de réception qui est visible <br><br>
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]:
```suggestion
>> Vous avez maintenant accès au contenu de votre compte e-mail. Par défaut, c'est votre boite de réception qui apparaît. <br><br>
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+> [!tabs]
+> **Etape 1**
+>>
+>> lorsque vous lancez l'application pour la première fois, la fenêtre ci-dessous s'affiche. Saisissez l'URL du serveur Zimbra OVHcloud `https://zimbra1.mail.ovh.net`, appuyez sur `Continuez`{.action} pour valider.
+>>
+>> ![zimbra_app](images/zimbra_app_connect01.png){.thumbnail .w-600 .h-600}
+>>
+> **Etape 2**
+>>
+>> Saisissez votre **adresse e-mail** Zimbra et le  **mot de passe** associé, appuyez ensuite sur `Connexion`{.action}
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]:
```suggestion
>> Saisissez votre **adresse e-mail** Zimbra et le  **mot de passe** associé, appuyez ensuite sur `Connexion`{.action}.
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+>>
+> **Etape 3**
+>>
+>> Vous avez maintenant accès au contenu de votre compte e-mail. Par défaut c'est votre boite de réception qui est visible <br><br>
+>>
+>> ![zimbra_app](images/zimbra_app_inbox01.png){.thumbnail .w-600 .h-600}
+
+> [!success]
+>
+> L'interface de l'application Zimbra est similaire au webmail. Si vous souhaitez obtenir plus de détails sur les fonctionnalités et paramètres, consulter notre guide [Utiliser le webmail Zimbra](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra)
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]:
```suggestion
> L'interface de l'application Zimbra est similaire au webmail. Pour plus de détails sur les fonctionnalités et paramètres, consultez notre guide « [Utiliser le webmail Zimbra](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra) ».
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+
+### Utiliser l'adresse e-mail
+
+Une fois l'adresse e-mail configurée, il ne reste plus qu’à l'utiliser ! Vous pouvez dès à présent envoyer et recevoir des messages.
+
+OVHcloud propose aussi une application web permettant d'accéder à votre adresse e-mail depuis un navigateur internet. Celle-ci est accessible à l’adresse [Webmail](/links/web/email). Vous pouvez vous y connecter grâce aux identifiants de votre adresse e-mail. Pour toute question relative à son utilisation, aidez-vous de notre guide [Utiliser le webmail Zimbra](/pages/web_cloud/email_and_collaborative_solutions/).
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]:
```suggestion
OVHcloud propose aussi une application web permettant d'accéder à votre adresse e-mail depuis un navigateur internet. Celle-ci est accessible via [ce lien](/links/web/email). Vous pouvez vous y connecter grâce aux identifiants de votre adresse e-mail. Pour toute question relative à son utilisation, consultez notre guide « [Utiliser le webmail Zimbra](/pages/web_cloud/email_and_collaborative_solutions/) ».
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+
+### Consulter une autre adresse e-mail Zimbra <a name="modify-settings"></a>
+
+L'application Zimbra ne permet de configurer qu'une adresse e-mail à la fois. Il faut donc se déconnecter du compte e-mail en cours d'utilisation pour pouvoir consulter un autre compte e-mail Zimbra.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]:
```suggestion
L'application Zimbra ne permet de configurer qu'une seule adresse e-mail à la fois. Il faut donc se déconnecter du compte e-mail en cours d'utilisation pour pouvoir consulter un autre compte e-mail Zimbra.
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+
+Dirigez-vous vers le menu « burger » dans le coin supérieur gauche de votre écran, puis sélectionnez l'icône de profil dans la partie supérieure de votre écran. Enfin, appuyez sur `Déconnexion`.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]:
```suggestion
Dirigez-vous vers le menu « burger » (☰) dans le coin supérieur gauche de votre écran, puis sélectionnez l'icône de profil dans la partie supérieure de votre écran. Enfin, appuyez sur `Déconnexion`.
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+
+![zimbra_app](images/zimbra_app_settings01.png){.thumbnail .w-600 .h-600}
+
+### Que faire si un écran blanc apparait lorsque je lance l'application ?
+
+Il s'agit d'un défaut de chargement lié à la connectivité au réseau, il est conseillé de patienter un instant pour s'assurer que l'application est eu le temps d'aller au bout de ça tentative de connexion.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]:
```suggestion
Il s'agit d'un défaut de chargement lié à la connectivité au réseau. Il est conseillé de patienter quelques instants, le temps que l'application termine sa tentative de connexion.
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+
+Si rien de ne passe, il est nécessaire de s'assurer que la connectivité réseau est correcte, puis fermer complètelment l'application et la relancer. Répéter l'opération autant de fois que nécessaire.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]:
```suggestion
Si rien de ne passe, vérifiez que la connectivité réseau de votre mobile est correcte, puis fermez complètement l'application et relancez-la. Répétez l'opération autant de fois que nécessaire.
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+
+Si le problème persiste, une réinstallation de l'application vous permettra de réinitialiser les paramètres pour éliminer un défaut de corruption du fichier de configuration de l'appiclation.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.fr-fr.md]:
```suggestion
Si le problème persiste, réinstallez l'application. Cela permettra de réinitialiser les paramètres et d'éliminer une potentielle corruption du fichier de configuration de l'application.
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/images/zimbra_app_connect02.png
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/images/zimbra_app_connect02.png:1]: Can you try to get that one in English?

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/meta.yaml [hunk 16]
```diff
@@ -0,0 +1,2 @@
+id: 8B22E20E-8B99-4346-8365-C651DA8FB40A
+full_slug: zimbra-mail-app-zimbra-android-ios
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/meta.yaml]:
```suggestion
full_slug: zimbra-mail-app-android-ios
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/images/zimbra_app_connect02.png
Comment (BrenKt) [reply pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/images/zimbra_app_connect02.png:1]: unfortunately i can't, i think the login page language is forced by the server, even with an english system

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.es-es.md [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+
+Es posible configurar las cuentas de correo electrónico de Zimbra en el cliente de correo que usted utilice. para que pueda utilizar su dirección de correo desde cualquier dispositivo. La aplicación Zimbra está disponible gratuitamente en Android e iOS
+
+**Descubra cómo configurar su cuenta Zimbra en la aplicación Zimbra.**
+
+> [!warning]
+>
+> OVHcloud pone a su disposición servicios cuya configuración, gestión y responsabilidad recaen sobre usted. Por lo tanto, usted deberá asegurarse de que estos funcionen correctamente.
+>
+> Esta guía le ayudará a realizar las tareas más habituales. No obstante, si necesita ayuda, le recomendamos que contacte con un [partner especializado](https://marketplace.ovhcloud.com/c/support-collaboration) o con el editor del servicio. Nosotros no podremos asistirle. Para más información, consulte el apartado «Más información» de esta guía.
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.es-es.md]: https://marketplace.ovhcloud.com/c/support-collaboration => available for all subs ?

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.es-es.md [hunk 18]
```diff
@@ -0,0 +1,109 @@
...
+- Disponer del nombre de usuario y la contraseña de la dirección de correo electrónico que quiera configurar.
+
+## Procedimiento
+
+### Configurar una dirección de correo electrónico
+
+Siga los pasos de instalación haciendo clic en las fichas siguientes:
+
+> [!tabs]
+> **Paso 1**
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.es-es.md]:
```suggestion
> **Etapa 1**
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.es-es.md [hunk 19]
```diff
@@ -0,0 +1,109 @@
...
+>>
+>> al iniciar la aplicación por primera vez, aparecerá la siguiente ventana. Introduzca la URL del servidor Zimbra OVHcloud `https://zimbra1.mail.ovh.net`, pulse `Continue`{.action} para validar.
+>>
+>> ![zimbra_app](images/zimbra_app_connect01.png){.thumbnail .w-600 .h-600}
+>>
+> **Paso 2**
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.es-es.md]:
```suggestion
> **Etapa 2**
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.es-es.md [hunk 20]
```diff
@@ -0,0 +1,109 @@
...
+>>
+>> Introduzca su **dirección de correo electrónico** Zimbra y la **contraseña** asociada y pulse `Conexión`{.action}.
+>>
+>> ![zimbra_app](images/zimbra_app_connect02.png){.thumbnail .w-600 .h-600}
+>>
+> **Paso 3**
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.es-es.md]:
```suggestion
> **Etapa 3**
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.it-it.md [hunk 21]
```diff
@@ -0,0 +1,110 @@
+---
+title: "Zimbra - Configurer son compte e-mail sur l'application mobile Zimbra"
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.it-it.md]: titre en FR

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.it-it.md [hunk 22]
```diff
@@ -0,0 +1,110 @@
...
+excerpt: "Découvrez comment configurer votre adresse e-mail Zimbra sur l'application mobile Zimbra disponible sur Android et iOS"
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.it-it.md]: Excerpt en FR

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.it-it.md [hunk 23]
```diff
@@ -0,0 +1,110 @@
...
+> **Importante**
+>
+> L'offerta Zimbra è un prodotto in fase beta.
+>
+> È disponibile solo per coloro che hanno compilato il [modulo di iscrizione alla beta](https://labs.ovhcloud.com/en/zimbra-beta/).
+>
+> Alcune delle funzionalità e dei limiti illustrati in questa guida potrebbero cambiare quando il prodotto viene immesso sul mercato.
+
+
+## Objectif
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.it-it.md]: Tout est en FR de la ligne 27 jusqu'à la fin

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.pt-pt.md [hunk 24]
```diff
@@ -0,0 +1,109 @@
+---
+title: "Zimbra - Configurar a sua conta de e-mail na aplicação móvel Zimbra"
+excerpt: "Descubra como configurar o seu endereço de e-mail Zimbra na aplicação móvel Zimbra disponível em Android e iOS"
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.pt-pt.md]:
```suggestion
excerpt: "Saiba como configurar o seu endereço de e-mail Zimbra na aplicação móvel Zimbra disponível em Android e iOS"
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.pt-pt.md [hunk 25]
```diff
@@ -0,0 +1,109 @@
...
+>>>
+> **Etapa 3**
+>>>
+>>> Já tem acesso ao conteúdo da sua conta de e-mail. Por predefinição, a sua caixa de entrada é apresentada <br><br>
+>>>
+>>>![zimbra_app](images/zimbra_app_inbox01.png){.thumbnail .w-600 .h-600}
+
+> [!success]
+>
+> A interface da aplicação Zimbra é semelhante ao webmail. Para mais pormenores sobre as funcionalidades e os parâmetros, consulte o guia « [Utilizar o webmail Zimbra](/pages/web_cloud/email_and_colaborative_solutions/mx_plan/email_zimbra) ».
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.pt-pt.md]:
```suggestion
> A interface da aplicação Zimbra é semelhante ao webmail. Para mais pormenores sobre as funcionalidades e os parâmetros, consulte o guia "[Utilizar o webmail Zimbra](/pages/web_cloud/email_and_colaborative_solutions/mx_plan/email_zimbra)".
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.pt-pt.md [hunk 26]
```diff
@@ -0,0 +1,109 @@
...
+
+### Utilizar o endereço de e-mail
+
+Depois de configurar o endereço de e-mail, já só precisa de o utilizar! Já pode enviar e receber mensagens no seu dispositivo.
+
+A OVHcloud também disponibiliza uma aplicação web que pode usar para aceder ao seu e-mail diretamente a partir do browser. Ela está acessível através de [este link](/links/web/email). Para aceder, só precisa dos dados de acesso do seu endereço de e-mail. Para qualquer questão relativa à sua utilização, consulte o guia « [Utilizar o webmail Zimbra](/pages/web_cloud/email_and_colaborative_solutions/) ».
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.pt-pt.md:77]:
```suggestion
A OVHcloud também disponibiliza uma aplicação web que pode usar para aceder ao seu e-mail diretamente a partir do browser. Ela está acessível através de [este link](/links/web/email). Para aceder, só precisa dos dados de acesso do seu endereço de e-mail. Para qualquer questão relativa à sua utilização, consulte o guia "[Utilizar o webmail Zimbra](/pages/web_cloud/email_and_colaborative_solutions/)".
```

**Review by Kazarus4 — COMMENTED** [review]
IT translations's missing

**Review by bs-ovh — APPROVED** [review]

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.es-es.md (hunk 17 above)
Comment (Y0Coss) [reply pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.es-es.md]: why don't you use `/links/partner` instead ?

**Pull request merged**